src/bootstrap/dist.rs

```diff
@@ -434,11 +434,12 @@ impl Step for Rustc {
                         .map_err(|err| format!("could not parse SOURCE_DATE_EPOCH: {}", err))
                         .unwrap();
 
-                    time::at(Timespec::new(epoch, 0))
+                    OffsetDateTime::from_unix_timestamp(epoch).unwrap()
                 })
-                .unwrap_or_else(|_| time::now());
+                .unwrap_or_else(|_| OffsetDateTime::now_utc());
```

Comment:
Used https://docs.rs/time/0.3.5/time/struct.OffsetDateTime.html#method.now_utc, but maybe https://docs.rs/time/0.3.5/time/struct.OffsetDateTime.html#method.now_local should be used instead?

Reply:
What did the previous function do? Let's try to match that.

Reply:
Looking at description at 0.1.x [version](https://docs.rs/time/0.1.44/time/fn.now.html), it says `Returns the current time in the local timezone`, so i guess it is.
From 0.3.6 (https://github.com/time-rs/time/blob/main/CHANGELOG.md#036-2022-01-20) `now_local` can be used again on more platforms.